web/web.py

```diff
@@ -2,22 +2,27 @@
 import pathlib
 import sys
 from decouple import config
+import numpy as np
 
 from flask import render_template, request
 
+
+
 PARENT_FOLDER = str(pathlib.Path(__file__).parent.parent.absolute())
 sys.path.insert(0, PARENT_FOLDER)
+from models.animals_classification.model import Predictor
 
 # flake8: noqa
 from logic import make_sentiment_prediction, make_disaster_prediction, compile_results, make_homepage_queryset, \
     make_category_projects_queryset
 # flake8: noqa
```

Comment:
For what do you need here noqa

Reply:
We need it because of flake 8. It gives error " module level import not at top of file".

Reply:
Turn off exact flake8 parameter at the top of the file.
With flake8: noqa you disable full flake8 for the whole file